app/helpers/icons_helper.rb

```diff
@@ -36,6 +36,10 @@ def op_icon(classnames, title: nil)
     %(<i class="#{classnames}" #{title} aria-hidden="true"></i>).html_safe
   end
 
+  def spot_icon(classnames, title: nil)
```

Comment:
I'd rather see us rework this interface to only accept the name of the icon, so entering `"enterprise-badge"` will output a `<span class="spot-icon spot-icon_enterprise-badge"></span>`

Reply:
Yes I agree that would be a lot easier to use 👍